pluto-plugins/plugins/logger/lib/src/main/java/com/pluto/plugins/logger/internal/DataModel.kt

```diff
@@ -21,12 +24,15 @@ internal sealed class Level(
 }
 
 @Keep
+@JsonClass(generateAdapter = true)
 internal data class LogData(
     val level: Level,
     val tag: String,
     val message: String,
-    val tr: Throwable? = null,
-    val stackTraceElement: StackTraceElement,
-    val eventAttributes: HashMap<String, Any?>? = null,
+    val tr: ExceptionData? = null,
+    val fileName: String,
```

Comment:
why do we need to remove `StackTraceElement`?

Reply:
On use StackTraceElement i have error "StackTraceElement requires explicit JsonAdapter to be registered", and i replace it on few primitiv fields

Reply:
this'll impact code readability, lets create a custom class to mimic StackTraceElement,
can start with `LogProcessor.stackTraceElement()`

Reply:
done